Vagrantfile.example

```diff
@@ -31,6 +34,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         domain.graphics_type = "spice"
         domain.memory = 2048
         domain.video_type = "qxl"
+        # Uncomment the following line if you would like to enable libvirt's unsafe cache
+        # mode. It is called unsafe for a reason, as it causes the virtual host to ignore all
+        # fsync() calls from the guest. Only do this if you are comfortable with the possibility of
+        # your development guest becoming corrupted (in which case you should only need to do a
+        # vagrant destroy and vagrant up to get a new one).
```

Comment:
A blank line would be good here to help highlight the code line that is meant to be uncommented in the text.